apps/src/storage/dataBrowser/EditTableRow.jsx

```diff
@@ -48,14 +51,26 @@ class EditTableRow extends React.Component {
   }
 
   handleSave = () => {
-    this.setState({isSaving: true});
-    const newRecord = _.mapValues(this.state.newInput, castValue);
-    FirebaseStorage.updateRecord(
-      this.props.tableName,
-      newRecord,
-      this.resetState,
-      msg => console.warn(msg)
-    );
+    if (experiments.isEnabled(experiments.APPLAB_DATASETS)) {
```

Comment:
nit / heads up: we might want to store this in a variable (`const inExperiment = experiments.isEnabled(experiments.APPLAB_DATASETS)`) or in a parent component's state since we check this value in so many places. 

i don't think it'll be a problem here, but we've run into performance issues in the past because we were checking for an experiment in the `render()` method of a component. this is obviously different, so i don't feel too concerned about it; this is mostly a heads up in case we notice a performance problem

Reply:
Oh that's good to know, thanks! Fixed!